dockers/docker-router-advertiser/radvd.conf.j2

```diff
@@ -10,9 +10,19 @@
 
 {# If our configuration has VLAN interfaces... #}
 {% if VLAN_INTERFACE %}
+{% set vlan_list = dict() %}
+{% for vlan_name in VLAN %}
```

Comment:
why we need to loop on VLAN table and check if `name==vlan_name`? I think to keep it same logic as before, can't we just need to loop VLAN_INTERFACE?

Reply:
VLAN table gives the vlan names, and VLAN_INTERFACE gives tuples like: "Vlan1000, fc02:1000::1" , "Vlan1000, fc02:2000::1". My intent is to create a prefix_list for every VLAN, and loop through VLAN_INTERFACE to add all ipv6 to corresponding VLAN's prefix_list.